source/docs/articles/local/git-faq.md

```diff
@@ -42,13 +43,13 @@ git push origin master
 
 For users who need any of the upstreams for Drupal distributions on Pantheon in order to resolve a conflict, we have added these URLs. If any of the upstream URLs are not working, please let us know.
 
-- [Open Academy](https://github.com/systemseed/openacademy-drops-7): https://github.com/systemseed/openacademy-drops-7
-- [Open Atrium](https://github.com/phase2/openatrium-drops-6.git): https://github.com/phase2/openatrium-drops-6.git
-- [Open Enterprise](https://github.com/levelten/openenterprise-drops-7.git): https://github.com/levelten/openenterprise-drops-7.git
-- [Open Outreach](https://github.com/nedjo/openoutreach-drops-7): https://github.com/nedjo/openoutreach-drops-7
-- [Open Publish](https://github.com/phase2/openpublish-drops-7.git): https://github.com/phase2/openpublish-drops-7.git
-- [Open Public](https://github.com/phase2/openpublic-drops-7.git): https://github.com/phase2/openpublic-drops-7.git
-- [Panopoly](https://github.com/populist/panopoly-drops-7.git): https://github.com/populist/panopoly-drops-7.git
+- [Open Academy](https://github.com/systemseed/openacademy-drops-7)

```

Comment:
Thes URLs are included to make it easier for users to copy/paste when resolving upstream conflicts locally. Let's leave them and consider removing `.git` from the hyperlinked URLs so that the links take users to the repo page instead of to download the repo.